### PR TITLE
Task config documentation

### PR DESCRIPTION
**Pull request by alxbenz**

### CODE REVIEW

File: config.js [hunk 1]
```diff
@@ -74,8 +70,6 @@ module.exports = {
 		]
 	},
 
-	cssmin: {},
```
Comment (janrembold) [inline config.js]: cssmin is still in use. why did you delete this?

File: README.md [hunk 2]
```diff
@@ -1,18 +1,308 @@
...
+   <Object> externalResources: {} //see external resources   
+   <Object> bowerResources: {} //see bower resources
+}
+```
+###### tasks
+You can toggle certain task in the build-framework. Every Task can be configurated with a boolean. 
+
+This list shows the configurable tasks as well as a brief description about what they do.
+```
+{
```
Comment (janrembold) [inline README.md]: I think this list is not complete, see https://github.com/frontend-framework/build-framework/blob/lazy-load-plugins/config.js#L49 for a more complete task list

File: config.js (hunk 1 above)
Comment (alxbenz) [reply config.js]: I don't think cssmin is still in use. I could not find any reference to it.
This got replaced by gulp-clean-css since it was deprecated.

File: README.md (hunk 2 above)
Comment (alxbenz) [reply README.md]: I added the missing tasks.

File: config.js (hunk 1 above)
Comment (janrembold) [reply config.js]: Right, sorry, I mixed it up with the cleanCss option
